### DISCUSSION TITLE
i have a problem with Automated Setup (New Project)

### DISCUSSION THREAD

**Discussion by GNUXDAR** [markshust/docker-magento#1134] [General]
Hello everyone, when I try to install a new project automatically, it gives me an error.

Note:
It doesn't install Magento.

![Captura desde 2024-04-15 16-40-01](https://github.com/markshust/docker-magento/assets/6179849/3bdb2c8b-4bbc-4eef-aa65-391c6f803504)

**Comment by YevhenZvieriev** [comment]
Hi @GNUXDAR 

It was resolved in #1133.

Do you have any questions, or this discussion may be closed?
